src/_pyinstaller_hooks_contrib/tests/test_libraries.py

```diff
@@ -561,3 +561,10 @@ def test_blspy(pyi_builder):
     pyi_builder.test_source("""
         import blspy
         """)
+
+
+@importorskip('flirpy')
+def test_flirpy(pyi_builder):
+    pyi_builder.test_source("""
+        import flirpy
+        """)
```

Comment:
`flirpy` has [empty top-level package](https://github.com/LJMUAstroecology/flirpy/blob/master/flirpy/__init__.py), so I don't think we're really testing anything here (i.e., the test passes even without the hook and without `flirpy/bin` being collected).

Reply:
okay, the usage in my actual application is:

`from flirpy.camera.lepton import Lepton
`

My thought was this was too specific, so tried to make the test case more general.  Open to suggestions on how you'd approach this.

Reply:
Specific is fine but maybe do several separate tests - one for each camera type? I'm worried that you may need a physical camera to run enough of `flirpy` to force it to try (and fail if it doesn't have the hook) to load these files in `flirpy/bin`.

Reply:
I imagine you need the actual camera to test to the extent when the lack of collected `bin` folder becomes a problem (i.e., `from flirpy.camera.lepton import Lepton` doesn't trigger an error, either). As such, I think dropping the test altogether might be the best idea here.

Reply:
Given that `flirpy/bin/find_camera.exe` is loaded [here](https://github.com/LJMUAstroecology/flirpy/blob/a094fdeb09c6f775fd82e7f0287b461ab792382d/flirpy/camera/lepton.py#L33) perhaps:
```python
from flirpy.camera.lepton import Lepton

print(Lepton.find_video_device())
```
would be enough?

Reply:
Indeed, that would do the trick. It returns `None` as opposed to raising exception when no camera is found, and `[WinError 2] The system cannot find the file specified` if `bin` is missing.

Reply:
Thanks folks - changes are pushed.